qemu/qmp.go

```diff
@@ -1604,3 +1609,14 @@ func (q *QMP) ExecuteQueryStatus(ctx context.Context) (StatusInfo, error) {
 
 	return status, nil
 }
+
+// ExecQomSet qom-set path property value
+func (q *QMP) ExecQomSet(ctx context.Context, path, property string, value uint64) error {
```

Comment:
Thanks @teawater  Could you add a unit test for this?

Reply:
@devimc Good catch.

Reply:
Done.